### PR TITLE
Feat: Add new table colors

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Use CSS variables for the colors of the new table in the Tokens page.

In this PR, add variables in the dark and light themes.

# Changes

* Add four new CSS variables for colors in light and dark themes.

# Screenshots

No changes.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@dskloetd as agreed with Artem, I added the new colors to gix-components.

File: src/lib/styles/themes/dark.scss [hunk 1]
```diff
@@ -10,6 +10,10 @@
   --text-text: var(--purple-900);
   --input-focus-background: var(--purple-dark-450);
   --button-primary: var(--purple-dark-accent);
+  --table-background: var(--purple-dark-550);
+  --table-header-background: var(--purple-dark-450);
+  --table-row-hover: var(--purple-dark-500);
+  --table-row: var(--purple-dark-550);
```
Comment (dskloetd) [inline src/lib/styles/themes/dark.scss:16]: Are these last 2 also backgrounds? Should we not add `background` to their names as well?

Comment (dskloetd) [inline src/lib/styles/themes/dark.scss:16]: Optionally you could point this one to the other one like this:
```
--table-row: var(--table-background);
```
But we can also wait with that until we add more layers.

Comment (lmuntaner) [reply src/lib/styles/themes/dark.scss:16]: I'd rather wait to add layers.

And those should probably be set by the designer.

**Comment by lmuntaner** [comment]
@dskloetd ready for another review

File: src/lib/styles/themes/dark.scss [hunk 2]
```diff
@@ -10,6 +10,10 @@
   --text-text: var(--purple-900);
   --input-focus-background: var(--purple-dark-450);
   --button-primary: var(--purple-dark-accent);
+  --table-background: var(--purple-dark-550);
+  --table-header-background: var(--purple-dark-450);
+  --table-row-background: var(--purple-dark-550);
+  --table-row-background-hover: var(--purple-dark-500);
```
Comment (dskloetd) [inline src/lib/styles/themes/dark.scss:16]: I think for background colors, `background` should be the suffix, so `hover` should come before it.
For example, there is also `--button-card-focus-background` where `focus` comes before `background`.

**Review by dskloetd — APPROVED** [review]

File: src/lib/styles/themes/dark.scss (hunk 2 above)
Comment (lmuntaner) [reply src/lib/styles/themes/dark.scss:16]: I don't have a strong opinion, but I agree that we should have consistent naming conventions.

File: src/lib/styles/themes/dark.scss (hunk 1 above)
Comment (lmuntaner) [reply src/lib/styles/themes/dark.scss]: These are the same name as in Figma, but let me ask Artem to change it. It makes sense to me.

**Pull request merged**